crates/dbs-boot/src/aarch64/mod.rs

```diff
@@ -4,21 +4,30 @@
 //! VM boot related constants and utilities for `aarch64` architecture.
 
 use vm_fdt::Error as VmFdtError;
-use vm_memory::GuestMemoryError;
+use vm_memory::{GuestMemoryError, GuestAddress, GuestMemory, Address};
 /// Magic addresses externally used to lay out aarch64 VMs.
 pub mod layout;
 
 /// FDT is used to inform the guest kernel of device tree information.
 pub mod fdt;
 
+/// Default (smallest) memory page size for the supported architectures.
+pub const PAGE_SIZE: usize = 4096;
+
 /// Errors thrown while configuring the Flattened Device Tree for aarch64.
-#[derive(Debug)]
+#[derive(Debug, thiserror::Error)]
 pub enum Error {
     /// Failure in creating FDT
+    #[error("create fdt fail")]
     CreateFdt(VmFdtError),
```

Comment:
Print the original error message.
```suggestion
    #[error("create fdt fail: {0}")]
    CreateFdt(#[source] VmFdtError),
```

Reply:
done, I have add #[from] attribute for the error.